integration/local-gadget/trace_exec_test.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	. "github.com/inspektor-gadget/inspektor-gadget/integration"
+	execTypes "github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/trace/exec/types"
+)
+
+func TestTraceExec(t *testing.T) {
+	t.Parallel()
+	ns := GenerateTestNamespaceName("test-trace-exec")
+
+	cmd := "while true; do date ; sleep 0.1; done"
+	// shArgs := []string{"/bin/sh", "-c", cmd}
+	dateArgs := []string{"/bin/date"}
+	sleepArgs := []string{"/bin/sleep", "0.1"}
+
+	traceExecCmd := &Command{
+		Name:         "TraceExec",
+		Cmd:          fmt.Sprintf("local-gadget trace exec -o json --runtimes=%s", *containerRuntime),
+		StartAndStop: true,
+		ExpectedOutputFn: func(output string) error {
+			expectedEntries := []*execTypes.Event{
+				// TODO: Enable this test once we can trace new cri-o containers.
```

Comment:
Ah interesting, does this mean we are losing some events because trace is being started after container creation. Right?

Reply:
Yep, based on how much time the gadget takes to start, we could be losing the very first events.